app/views/catalog/_catalog_listing.html.erb

```diff
@@ -1,17 +1,18 @@
 <% eq_models_by_category.each do |category, equipment_models| %>
-      <% unless equipment_models.nil? %>
-        <h2 class="catalog">
-          <%= link_to category.name, category_equipment_models_path(category) %>
-        </h2>
-        <div class="thumbnails row">
-        <% equipment_models.each do |equipment_model| %>
-          <div class="col-md-4">
-          <%= render partial: 'catalog/equipment_model_div', locals: { equipment_model: equipment_model,
-                                                               availability_hash: availability_hash,
-                                                               available_string: available_string,
-                                                              } %>
-          </div>
-        <% end %>
-      </div>
-      <% end %>
-    <% end %>
+<% unless equipment_models.nil? %>
```

Comment:
The indentation here is wrong; each successive Ruby / ERB code level should be indented, as should be any HTML within those blocks.